src/components/card/_macro.njk

```diff
@@ -10,7 +10,7 @@
       {%- if params.url -%}
```

Comment:
Have a word with Dina about whether there is any need to use the card component without a link. Because in the docs it says `url` is required. If it is required theres no need for these ifs at all and then in the css change we should just be able to remove the `margin-bottom` line. If there is a need for it then we should update the documentation to say its not required.

Reply:
I think the IDS are currently using the card component in their homepage without the urls. I think it might be best to update the documentation and make them optional.

Reply:
In that case, if we are allowing this we should add an example where there is no url. Then we can cover that case with the visual tests

Reply:
Oh wait a second! I think I confused the url with the itemlist param. No you always have a link as a header. Please ignore my comment and go with Richard first suggestion. Speak with Dina and then the rest lol